Add support for multiple claims per paper

anishathalye opened the pull request:
As the title says.

This change gets rid of sorting on threat model / dataset (it wasn't all that useful anyways, in my opinion; filtering still works). It also adds a sticky header.

This change also includes an example of multiple claims per paper (from #4).

cc @max-andr

anishathalye committed:
Add sticky header

anishathalye committed:
Add support for multiple claims per paper

anishathalye committed:
Add additional claims

anishathalye committed:
Fix merging of venue column

anishathalye committed:
Add certified accuracy

anishathalye commented:
And here is a screenshot, so people don't have to run this themselves to see what it looks like:

<img width="1166" alt="Screen Shot 2019-05-30 at 3 14 42 PM" src="https://user-images.githubusercontent.com/3526486/58658082-ba8a5a00-82ed-11e9-9e35-3f1e8a0e8f41.png">

max-andr commented:
Looks nice in my opinion. Maybe adding a bit more horizontal space to column "Claims" would improve the readability of the table.

One small thing regarding our numbers (from the screenshot) - on MNIST we have the results not on 1k, but on 10k points (full test set). And the empirical adversarial accuracy is 96.42% instead of 96.4%. Just to be precise in the claims :)

anishathalye committed:
Fix claims

See
https://github.com/robust-ml/robust-ml.github.io/pull/8#issuecomment-497462818.

anishathalye commented:
Oops, must have missed that. Yes, we certainly care about being precise in the claims -- fixed :)

I tried playing with the column spacing, but I couldn't get it to look much better. I'm going to merge this as-is, and if anyone can figure out how to make the table layout better, then a separate pull request would be appreciated.
